server/lobbyconnection.py

```diff
@@ -55,27 +58,30 @@ def __init__(
         self,
         games: GameService,
         players: PlayerService,
+        nts_client: Optional[TwilioNTS],
         geoip: GeoIpService,
         matchmaker_queue: MatchmakerQueue
     ):
         self.geoip_service = geoip
         self.game_service = games
         self.player_service = players
+        self.nts_client = nts_client
+        self.coturn_generator = CoturnHMAC()
```

Comment:
Why do we create a new `CoturnHMAC` for every connection, but only have a single `TwilioNTS`? Couldn't `CoturnHMAC` just be a singleton as well?